### DISCUSSION TITLE
Trouble with device

### DISCUSSION THREAD

**Discussion by fladmas** [bisand/node-red-contrib-panasonic-comfort-cloud#2] [Q&A]
groups works perfectly including getting the guid;   payload[0]._devices[0].guid

But when entering the guid as a text msg.payload in a new inject it just times out.

Any ideas as what I am doing wrong?

**Comment by fladmas** [comment]
please, any ideas would be wellcome

**Comment by bisand** [comment]
I have updated the underlying library and made some small changes to the logic, and it seems to be working better now. Try to update the component to 0.2.0 and try again. Please let me know if it succeeded or not.

Thanks! :)

**Reply by fladmas** [reply]
cool, I will try. Thanks - much appreciated!!
